### ISSUE TITLE
Accessibility: 'Select box' without a description

### ISSUE THREAD

**Issue by banksashleym** [JacksonCollege/jc-theme-2015#218]
Also getting an error on the homepage 'Select box' without a description for the following code
`<div class="slick-track" style="opacity: 1; width: 5334px; z-index: 2147483646;" role="listbox">...</div>`

Suggested fix:
1.3.1 Info and Relationships

Drop down menus (select boxes or role="listbox") should always have a description that is explicitly associated with the field to make sure that users of assistive technologies will also know what the field is for.

If the drop down menu (select box or role="listbox") has a visible description indicating the purpose of the menu, this description should be explicitly associated to the input drop down menu either as a HTML label or using the WAI-ARIA attribute 'aria-labelledby'.

If it is not possible to add a visible description, either add a mouseover text ('title' attribute) to the drop down menu or create an invisible label using the WAI-ARIA attribute 'aria-label'.

**Comment by joelworsham** [comment]
In a recent update to Slick (plugin for slideshow), `listbox` was replaced with `tablist`. Could you check this again? THanks!

**Comment by banksashleym** [comment]
Yes, this seems to be fixed :) Thanks!
